docs/fsharp/using-fsharp-on-azure/queue-storage.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Get started with Azure Queue storage using F# 
+description: Azure Queues provide reliable, asynchronous messaging between application components. Cloud messaging enables your application components to scale independently.
+keywords: visual f#, f#, functional programming, .NET, .NET Core, Azure
+author: sylvanc
+manager: jbronsk
+ms.date: 09/20/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+ms.assetid: 70dc554c-8f4d-42a7-8e2a-6438657d012a
+---
+
+# Get started with Azure Queue storage using F# #
+
+Azure Queue storage provides cloud messaging between application components. In designing applications for scale, application components are often decoupled, so that they can scale independently. Queue storage delivers asynchronous messaging for communication between application components, whether they are running in the cloud, on the desktop, on an on-premises server, or on a mobile device. Queue storage also supports managing asynchronous tasks and building process work flows.
+
+### About this tutorial
+
+This tutorial shows how to write F# code for some common tasks using Azure Queue storage. Tasks covered include creating and deleting queues and adding, reading, and deleting queue messages.
+
+For a conceptual overview of queue storage, please see [the .NET guide for queue storage](https://azure.microsoft.com/en-us/documentation/articles/storage-dotnet-how-to-use-queues/).
+
+## Prerequisites
+
+To use this guide, you must first [create an Azure storage account](https://azure.microsoft.com/en-us/documentation/articles/storage-create-storage-account/).
+You'll also need your storage access key for this account.
+
+## Create an F# Script and Start F# Interactive
+
+The samples in this article can be used in either an F# application or an F# script. To create an F# script, create a file with the `.fsx` extension, for example `queues.fsx`, in your F# development environment.
+
+Next, use a [package manager](package-management.md) such as [Paket](https://fsprojects.github.io/Paket/) or [NuGet](https://www.nuget.org/) to install the `WindowsAzure.Storage` package and reference `WindowsAzure.Storage.dll` in your script using a `#r` directive.
+
+### Add namespace declarations
+
+Add the following `open` statements to the top of the `queues.fsx` file:
+
+[!code-fsharp[QueueStorage](../../../samples/snippets/fsharp/azure/queue-storage.fsx#L1-L3)]
+
+### Get your connection string
+
+You'll need an Azure Storage connection string for this tutorial. For more information about connection strings, see [Configure Storage Connection Strings](https://azure.microsoft.com/en-us/documentation/articles/storage-configure-connection-string/).
+
+For the tutorial, you'll enter your connection string in your script, like this:
+
+[!code-fsharp[QueueStorage](../../../samples/snippets/fsharp/azure/queue-storage.fsx#L9-L13)]
```

Comment:
I think this snippet shouldn't show the commented out `CloudConfigurationManager.GetSetting` code, it's already shown in the snippet.